src/backend/Dockerfile

```
@@ -3,10 +3,13 @@ FROM quay.io/azavea/django:3.2-python3.10-slim
 RUN mkdir -p /usr/local/src/backend
 WORKDIR /usr/local/src/backend
 
-COPY requirements.txt /usr/local/src/backend/
+COPY /src/backend/requirements.txt /usr/local/src/backend/
```

```suggestion
COPY ./src/backend/requirements.txt .
```

I think it's possible to use `.` for the destination because we've specified `WORKDIR /usr/local/src/backend` up above.

---

This worked! Will edit.